tools/buildHeaders/bin/makeHeaders

```diff
@@ -3,5 +3,5 @@
 python3 bin/makeExtinstHeaders.py
 
 cd ../../include/spirv/unified1
-../../../tools/buildHeaders/build/install/bin/buildSpvHeaders -H spirv.core.grammar.json
+../../../tools/buildHeaders/build/buildSpvHeaders -H spirv.core.grammar.json
```

Comment:
Is this valid for Windows builds?  In that case it might be in a "Release" directory, or whatever the configuration name is.

Reply:
Not sure, I didn't test Windows. I don't want to break anybody's workflow so I'll close this PR and suggest https://github.com/KhronosGroup/SPIRV-Headers/pull/222 instead (changes the README and adds simple CI).